source/write-to-mongodb.txt

```diff
@@ -28,6 +28,18 @@ Write to MongoDB
 
          .. include:: /scala/write-to-mongodb.txt
 
+.. warning::
+
+   If you specify the ``overwrite`` save mode, the connector drops the target
```

Comment:
Q: is there a place in the docs that describes all of the mode options and how to set them?

Reply:
Not that I could find. The Spark docs walk a difficult line between native Spark behavior and what our connector adds to it. The write modes are technically [native Spark.](https://sparkbyexamples.com/spark/spark-write-modes-explained/)